images/favicons/safari-pinned-tab.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Ideally this should be compressed to reduce loading times.

Reply:
Ok